numba/tests/test_unicode.py

```diff
@@ -1277,6 +1277,24 @@ def pyfunc(x):
             self.assertEqual(pyfunc(*args), cfunc(*args),
                              msg='failed on {}'.format(args))
 
+    def test_isalpha(self):
+        def pyfunc(x):
+            return x.isalpha()
+
+        cfunc = njit(pyfunc)
+        # Samples taken from CPython testing:
+        # https://github.com/python/cpython/blob/201c8f79450628241574fba940e08107178dc3a5/Lib/test/test_unicode.py#L630-640    # noqa: E501
```

Comment:
```suggestion
        # https://github.com/python/cpython/blob/201c8f79450628241574fba940e08107178dc3a5/Lib/test/test_unicode.py#L630-L640    # noqa: E501
```
This syntax is invalid, `L` prefix needed ahead of both items in the range.